### ISSUE TITLE
RC Test: Ohms Law 1.4.0-rc.1

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/QA#313]
@emily-phet, @terracoda, @zepumph Ohms Law 1.4.0-rc.1 is ready for RC testing. This release candidate follows #289. This release will include sound and accessibility so please include tests related to those features. It should also include testing for iOS + Safari + VoiceOver, so please test that platform as well. Document issues in https://github.com/phetsims/ohms-law/issues and link to this issue.

Assigning to @ariel-phet for prioritization.

<details>
<summary><b>General RC Test</b></summary>

<h3>What to Test</h3>

- Click every single button.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
- If there is sound, make sure it works.
- Make sure you can't lose anything.
- Play with the sim normally.
- Try to break the sim.
- Test all query parameters on all platforms. (See [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md) for a list of query parameters.)
- Download HTML on Chrome and iOS.
- Make sure the iFrame version of the simulation is working as intended on all platforms.
- Make sure the XHTML version of the simulation is working as intended on all platforms.
- Complete the test matrix.
- Don't forget to make sure the sim works with Legends of Learning.
- Check [this](https://docs.google.com/spreadsheets/d/1umIAmhn89WN1nzcHKhYJcv-n3Oj6ps1wITc-CjWYytE/edit#gid=0) LoL spreadsheet and notify AR if it not there.
- If this is rc.1 please do a memory test.

<h3>Link(s)</h3>

- **[Simulation](https://phet-dev.colorado.edu/html/ohms-law/1.4.0-rc.1/phet/ohms-law_en_phet.html)**
- **[iFrame](https://phet-dev.colorado.edu/html/ohms-law/1.4.0-rc.1/phet/ohms-law_en_iframe_phet.html)**
- **[XHTML](https://phet-dev.colorado.edu/html/ohms-law/1.4.0-rc.1/phet/xhtml/ohms-law_all.xhtml)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1wcz2eUCsY6P5CPzmkKuWG7mjNrmqc0TzlVOy3fTmOt4/edit#gid=733912604)**
- **[Legends of Learning Harness](https://developers.legendsoflearning.com/public-harness/index.html?startPayload=%7B%22languageCode%22%3A%22en%22%7D)**

<hr>

</details>

<details>
<summary><b>Accessibility (a11y) RC Test</b></summary>

<h3>What to Test</h3>

- Specific instructions can be found below.
- Make sure the a11y feature doesn't negatively affect the sim in any way.
- Test all possible forms of input.
  - Test all mouse/trackpad inputs.
  - Test all touchscreen inputs.
  - Test all keyboard navigation inputs (if applicable).
  - Test all forms of input with a screen reader (if applicable).

<h3>Link(s)</h3>

- **[a11y View](https://phet-dev.colorado.edu/html/ohms-law/1.4.0-rc.1/phet/ohms-law_a11y_view.html)**
- **[Simulation](https://phet-dev.colorado.edu/html/ohms-law/1.4.0-rc.1/phet/ohms-law_en_phet.html)**
- **[Test Matrix](https://docs.google.com/spreadsheets/d/1wcz2eUCsY6P5CPzmkKuWG7mjNrmqc0TzlVOy3fTmOt4/edit#gid=733912604)**

<h3>Screen Readers</h3>

This sim supports screen readers. If you are unfamiliar with screen readers, please ask Katie to introduce you to screen readers. If you simply need a refresher on screen readers, please consult the [QA Book](link), which should have all of the information you need as well as a link to a screen reader tutorial made by Jesse. Otherwise, look over the a11y view before opening the simulation. Once you've done that, open the simulation and make sure alerts and descriptions work as intended.

<h3>Platforms and Screen Readers to Be Tested</h3>

- [x] Windows 10 + Latest Chrome + Latest JAWS
- [x] Windows 10 + Latest Firefox + Latest NVDA
- [x] macOS + Safari + VoiceOver
- [x] iOS + Safari + VoiceOver

<h3>Critical Screen Reader Information</h3>

We are tracking known screen reader bugs in [this Google Document](https://drive.google.com/open?id=12kTs-g-iKEIH1dyG7Q41_W_oNL4gUKbkW-IQgZjMUBw). If you find a screen reader bug, please check it against this list.

<h3>Keyboard Navigation</h3>

This sim supports keyboard navigation. Please make sure it works as intended on all platforms by itself and with a screen reader.

<h3>Final Requests</h3>

- [x] If this sim is in [this list](https://github.com/phetsims/perennial/blob/master/data/accessibility), make an issue in [website](https://github.com/phetsims/website) to ask if PhET research page links need updating. Please assign to @terracoda and @emily-phet.

<hr>

</details>

<details>
<summary><b>FAQs for QA Members</b></summary>

<br>
  
  

  <details>
  <summary><i>There are multiple tests in this issue... Which test should I do first?</i></summary> 
  
  Test in order! Test the first thing first, the second thing second, and so on.

  </details>

  <br>

  

  <details>
  <summary><i>How should I format my issue?</i></summary>
  
  Here's a template for making issues:

      <b>Test Device</b>

      blah

      <b>Operating System</b>

      blah

      <b>Browser</b>

      blah

      <b>Problem Description</b>

      blah

      <b>Steps to Reproduce</b>

      blah

      <b>Visuals</b>

      blah

      <details>
      <summary><b>Troubleshooting Information</b></summary>

      blah

      </details>

  </details>

  <br>

  

  <details>
  <summary><i>Who should I assign?</i></summary>
  
  We typically assign the developer who opened the issue in the QA repository.

  </details>
  
  <br>

  

  <details>
  <summary><i>My question isn't in here... What should I do?</i></summary>
  
  You should:

  1. Consult the [QA Book](https://github.com/phetsims/QA/blob/master/doc/qa-book.md).
  2. Google it.
  3. Ask Katie.
  4. Ask a developer.
  5. Google it again.
  6. Cry.

  </details>

<br>

<hr>

</details>

**Comment by jessegreenberg** [comment]
@terracoda @jbphet @emily-phet can you also please take a look at this version and let me know if the credits need to be updated?

**Comment by terracoda** [comment]
@jessegreenberg, there's a comment from @ariel-phet in https://github.com/phetsims/ohms-law/issues/54#issuecomment-313294755 that Kathy needs to be added to the Team list and a question about whether Martin needs to be mentioned.

In addition, I am aware of:
- Emily Moore, Taliesin Smith should added to Team, and
- Please add a Sound Design item with Ashton Morris and possibly Mike Winters (@emily-phet, please verify who needs to be put on the Sound Design list).

Regarding Software Developers or other Team members, I can't remember who has worked on this sim. Haven't both you and @zepumph worked on this sim?

Any more for QA?

**Comment by KatieWoe** [comment]
Myself and Liam at least.  We'll see who else does work on it during this rc.

**Comment by KatieWoe** [comment]
Memory Test Results
Start:20.5
1: 24.5
2: 25
3: 25.4
4: 25.6
5: 25.8
6: 26.1
7: 26.3
8: 26.5
9: 26.6
10: 26.6

**Comment by jessegreenberg** [comment]
Thanks for pointing out that issue @terracoda. @zepumph is the responsible dev and did the sim a11y implementation so he should be added to credits. I don't feel that I contributed much to this sim specifically outside of common code work.

**Comment by jessegreenberg** [comment]
Sorry everyone, lets continue with credits discussion in https://github.com/phetsims/ohms-law/issues/54#issuecomment-488456482

**Comment by KatieWoe** [comment]
QA is done

**Comment by jessegreenberg** [comment]
Excellent, thank you @KatieWoe.
